composer.json

```diff
@@ -32,7 +32,6 @@
     "composer/installers": "^1.2",
     "cweagans/composer-patches": "^1.6.5",
     "drupal-composer/drupal-scaffold": "^2.5",
-    "drupal/admin_toolbar": "^1",
```

Comment:
It should affect lock file and ++ to remove

Reply:
lock file not versioned here, so no problem)